app/helpers/flood_risk_engine/application_helper.rb

```diff
@@ -55,9 +55,5 @@ def page_title(title)
 
       title
     end
-
-    def submit_button_text(_enrollment)
```

Comment:
I removed it because I think just having `t("global.continue")` in the places where it is needed is clearer and more concise. The method is only used in two places. Also, have a look at app/views/flood_risk_engine/enrollments/steps/show.html.erb within this PR.